helper-cli/src/main/kotlin/commands/packagecuration/SplitCommand.kt

```diff
@@ -26,11 +26,16 @@ import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.required
 import com.github.ajalt.clikt.parameters.types.file
 
+import org.ossreviewtoolkit.helper.common.createBlockYamlMapper
 import org.ossreviewtoolkit.helper.common.getSplitCurationFile
+import org.ossreviewtoolkit.helper.common.wrapAt
 import org.ossreviewtoolkit.model.PackageCuration
 import org.ossreviewtoolkit.model.readValue
-import org.ossreviewtoolkit.model.writeValue
 import org.ossreviewtoolkit.utils.expandTilde
+import org.ossreviewtoolkit.utils.safeMkdirs
+
+// Wrap at column 120 minus 6 spaces of indentation.
+private const val COMMENT_WRAP_COLUMN = 120 - 6
```

Comment:
@fviernau I added this as it is required for our use case.
Do you think it makes sense to add a flag to enable / disable the block comments?

Reply:
In the long term I couldn't see a reason why this wrapping shouldn't be done by all helper cli commands which output YAML consistently. I wouldn't add a flag for now. Later on may considering introducing a helper-cli config file for such things.